File: QuanLyPhongKham.Server/src/QuanLyPhongKham.DTOs/Responses/DanhMucThuocResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyPhongKham.DTOs.Responses
{
    public class DanhMucThuocResponse
    {
        public Guid MaThuoc { get; set; }
        public string TenThuoc { get; set; }
        public decimal DonGia { get; set; }
        public string? ChongChiDinh { get; set; }
    }
}

File: QuanLyPhongKham.Client/src/interfaces/responses/DanhMucThuocResponse.js
/**
 * @typedef {Object} DanhMucThuocResponse
 * @property {string} maThuoc
 * @property {string} tenThuoc
 * @property {number} donGia
 * @property {string} chongChiDinh
 */

export const createDanhMucThuocResponse = (
  maThuoc = "",
  tenThuoc = "",
  donGia = 0,
  chongChiDinh = "",
) => ({
  maThuoc,
  tenThuoc,
  donGia,
  chongChiDinh,
});
